odoo/addons/base/models/assetsbundle.py

```diff
@@ -678,8 +676,42 @@ def get_command(self):
 
 
 class ScssStylesheetAsset(PreprocessedCSS):
+    @lazy_property
+    def bootstrap_path(self):
+        return get_resource_path('web', 'static', 'lib', 'bootstrap', 'scss')
+    @lazy_property
+    def bootstrap_components_path(self):
+        return get_resource_path('web', 'static', 'lib', 'bootstrap', 'scss', 'bootstrap')
+    precision = 8
+    output_style = 'expanded'
+
+    def get_compiler(self):
+        if libsass is None:
+            return super(ScssStylesheetAsset, self).get_compiler()
+
+        def libsass_wrapper(source):
```

Comment:
why a closure ?

Reply:
Because we can't convert exceptions using functools.partial, and I didn't want to change the entre protocol away from the get_ thing, though looking at the way it's actually used it would probably be possible.

Reply:
I think we should change the protocol into simple functions and get rid of the get_  stuff.

Reply:
done